.dockerignore

```diff
@@ -1,2 +1,6 @@
 node_modules
 npm-debug.log
+test
+data
+codeship-*
+*.md
```

Comment:
I noticed these files were being built into the image and added them to the `.dockerignore`